web/modules/custom/dpl_react_apps/dpl_react_apps.routing.yml

```diff
@@ -0,0 +1,14 @@
+dpl_react_apps.search_result:
+  path: "/search"
+  defaults:
+    _controller: '\Drupal\dpl_react_apps\Controller\DplReactAppsController::search'
+  requirements:
+    _permission: "access content"
+
+dpl_react_apps.material_page:
+  path: "/material/{pid}"
+  defaults:
+    _controller: '\Drupal\dpl_react_apps\Controller\DplReactAppsController::material'
+  requirements:
+    _permission: "access content"
+    pid: "[0-9]+-[a-z]+:[0-9]+"
```

Comment:
Consider updating this pattern. 

I understand the intention but this may be too strict. Example: [On beta.bibliotek.dk we see work ids starting with `work-of`](https://beta.bibliotek.dk/materiale/harry-potter-og-de-vises-sten_joanne-k-rowling/work-of%3A870970-basis%3A22629344). I expect that we will need to support such pids ourselves at well. They will not work with the current pattern.

Reply:
We had a longer talk about this. :) For now we stick to this version and adress future problems if they should arise. There is pro's and con's with every solution. And no seemingly "right" choice.